existing-communities/README.md

```diff
@@ -1 +1,47 @@
+A key factor of an ecosystem is stability. In a stable ecosystem{{"EcosysSDef" | cite}}, each species acts in such a way that they directly enable or inderectly do not interfere with the survival of another species. . An ecosystem will remain stable as long as the system is able to deal with outside disturbances.
+
+### Intentional Communities
+Intentional communities are man-made communities, where its members attempt to realise a common alternative way of life outside of mainstream society{{"meijering2007intentional" | cite}}. Although there is no one list of criteria to characterise them, the most cited criteria are:
+
+1. No bonds by familial relationships only.
+2. A minimum of three to five adult members.
+3. Members join voluntarily
+4. Geographical and psychological separation from mainstream society.
+5. A common ideology that is adhered to by all members.
+6. Sharing of (a part of) one's property.
+7. The interest of the group prevails over individual interests.
+
+Not all intentional communities fulfill all of these criteria but they give a good idea of what is and what is not a intentional community. Intentional communities can be quite diverse. To get a better understanding of the different ways an intentional community can work, they can be split into four groups: ecological, communal, religious and practical communities{{"meijering2007intentional" | cite}}.
+
+**Ecological Communities**  
+Ecological communities are communities that withdraw to remote locations in an effort to live according to their own ideals. They actively separate themself from society and aim to be self-sufficient. The separation is what identifies this type of intentional communities{{"meijering2007constructing" | cite}}{{"meijering2007intentional" | cite}}.  
+Ecological communities can manifest themself on different scales. It can vary from a group living together in a house (cohousing) to a whole village (ecovillages){{"sanguinetti2012design" | cite}}, but all have a support-structure and cooperate to achieve the ideals of the community{"svensson2002ecovillage" | cite}.
+
+**Communal Communities**  
+Communal communities center their ideals around the contact between communities members{{"meijering2007intentional" | cite}}. This means that the platform (chat-groups, website, etc.) or place (offices, pub, etc.) plays a central key in this type of community. The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.
```

Comment:
`The place or subject of the platform focuses on the particular characteristic that the members have in common, like a cooking-forum for people who like cooking or a chat-group for parents whos kids are in the same class.` ==> `The place or subject of the platform focuses on the particular characteristic that the members have in common. Examples of such communities are a cooking-forum for people who like cooking or a chat-group for parents whos**e** kids are in the same class.`